homeassistant/components/nest/__init__.py

```diff
@@ -347,9 +346,46 @@ async def get(
             return self._json_error(
                 f"No event found for event_id '{event_token}'", HTTPStatus.NOT_FOUND
             )
-        return web.Response(body=media.contents, content_type=media.content_type)
+        return await self.handle_media(media)
+
+    async def handle_media(self, media: Media) -> web.StreamResponse:
```

Comment:
Should this be an `@abstractmethod` to force children to implement it?

Reply:
Good catch. I have some followups touching this method and will address. 👍🏼